### ISSUE TITLE
createFont and tamagui.styled is not a function (it is undefined) Expo router v3

### ISSUE THREAD

**Issue by juniorogaa** [tamagui/tamagui#2168]
### Current Behavior

I'm changing my app's navigation from react-navigation to expo-router v3.
I did all the configuration correctly, and it was already working. However, when I add the Tabs, I receive this message and I've done everything and I can't resolve it.

<img width="300" alt="image" src="https://github.com/tamagui/tamagui/assets/96478892/608f0684-de59-48e4-9057-a62573ebc342">

<img width="300" alt="image" src="https://github.com/tamagui/tamagui/assets/96478892/f21b58fd-2b3a-4c9b-a6d9-ae70f70e8a91a">

app/_layout.tsx

<img width="800" alt="image" src="https://github.com/tamagui/tamagui/assets/96478892/c2dafdab-780b-4e8a-8395-796b4d18d8fa">

app/(tabs)/_layout.tsx
<img width="800" alt="image" src="https://github.com/tamagui/tamagui/assets/96478892/f29b16b2-22a7-4609-ba3f-d126d0c49117">

### Tamagui Version

```markdown
1.88.17
```

### Platform (Web, iOS, Android)

```markdown
Android, iOS, Web
```

### Reproduction

```markdown
Create an app with expo 50.0.4, tamagui 1.88.17 with expo-router v3, using tabs layout.
```

**Comment by likeSo** [comment]
🙋This issue will also happen when following the official [Expo Guide](https://tamagui.dev/docs/guides/expo)
After `yarn add @tamagui/metro-plugin` and editing `metro.config.js` file, i got this problem too.

**Comment by juniorogaa** [comment]
> 🙋This issue will also happen when following the official [Expo Guide](https://tamagui.dev/docs/guides/expo) After `yarn add @tamagui/metro-plugin` and editing `metro.config.js` file, i got this problem too.

Did you manage to fix it in any way?

**Comment by Imtiajrex** [comment]
Anyone found a solution to this yet?

**Comment by natew** [comment]
The starter kit works for me today with tabs: `npm create tamagui@latest` and select Expo Router.

Can you help me with a simple reproduction and I can probably fix it somewhat quickly, or at least forward it to Metro/Expo.

**Comment by natew** [comment]
In the latest version of tamagui the metro plugin you can `disableCSSInterop` just to test if that plugin is causing any issue.

**Comment by natew** [comment]
I just tested by making a new app using:

```
npx create-expo-app@latest --template tabs@50
```

And followed the instructions, minus adding the dependencies everything is working:

https://github.com/natew/example-expo-router

![CleanShot 2024-02-02 at 04 53 41@2x](https://github.com/tamagui/tamagui/assets/12100/fc967bcb-bf17-4a7e-acbb-058fd17455e1)
![CleanShot 2024-02-02 at 04 54 08@2x](https://github.com/tamagui/tamagui/assets/12100/b3046243-41ff-40a5-a0e0-44085d375d50)

You actually don't even need the metro setup just for native, it only makes web more performant.

**Comment by gorbypark** [comment]
Through some trial and error I have figured out that the name of the css file, `tamagui.css` is causing the issue (I believe).  I don't know the underlying cause, but a temporary fix is to change the css output in `metro.config.ts` to `outputCSS: './tamagui-output.css'` and then update the corresponding import in `app/_layout.tsx`.  It would be nice if anyone else experiencing the issue could confirm this indeed does fix it for you as well.

**Comment by junioroga** [comment]
I made a repo here -> https://github.com/junioroga/test-tamagui

just you need run yarn to install packages and run the app.

**Comment by natew** [comment]
Your versions aren't all aligned ^ which causes some issues, and your yarn version is a bit old to make the upgrade easy. Looking into adding a doctor command to tamagui that checks for versions being in sync.

**Comment by junioroga** [comment]
I updated yarn version, changed all packages to ˆ. but the problem still :(

https://github.com/junioroga/test-tamagui

**Comment by natew** [comment]
Hm so importing core works, importing tamagui and it's just an empty Object. So far don't see what's causing that but checking...

**Comment by natew** [comment]
Ah ok found the issue! It is `tamagui`, by having `tamagui.css` at the root for some reason Metro is trying to resolve it instead of the tamagui import. Looking into what I can do here.

**Comment by natew** [comment]
This would be a good issue to open on the Expo repo if anyone wants to, just letting them know `isCSSEnabled` causes Metro to prefer a css file over a node module, when it should in fact get the node module.

**Comment by natew** [comment]
Closing as we have a solution for now, and once Expo fixes its good https://github.com/expo/expo/issues/26580

**Comment by gorbypark** [comment]
> Closing as we have a solution for now, and once Expo fixes its good [expo/expo#26580](https://github.com/expo/expo/issues/26580)

Is the solution for now just naming the css to something other than tamagui.css?

**Comment by natew** [comment]
Yes. I can probably figure out a patch in the metro plugin but the better solution is probably in the expo default config.
